hazelcast-enterprise/Dockerfile

```diff
@@ -37,7 +37,9 @@ COPY *.sh *.yaml *.jar *.properties ${HZ_HOME}/
 
 # Install
 RUN echo "Installing new APK packages" \
-    && apk add --no-cache openjdk11-jre bash curl \
+    && apk update \
+    && apk add --no-cache openjdk15-jre --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing \
```

Comment:
Do we want to use the `testing` repository? Seems like something not production-ready. Plus, what are the benefits for upgrading to JRE15?

Reply:
That's literally what I asked in the PR description :)

G1 was greatly improved in JDK15.

Reply:
https://github.com/hazelcast/hazelcast-docker/pull/204#issuecomment-737737676